contract/Driver/ClientInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\StorageDriver\Contract\Driver;
+
+use Google\Protobuf\Internal\Message;
+
+interface ClientInterface
+{
+    /**
+     * @param string[] $features
+     * @return mixed
+     */
+    public function runCommand(
+        string $backend,
```

Comment:
tu ten `$backend` oznacuje co ? je to v repe `*-teradata` tak mi pride ze by to tu nemuselo byt

Reply:
odstranil sem